dss/events/handlers/index.py

```diff
@@ -218,9 +221,21 @@ def notify(subscription_id: str, subscription: dict, bundle_id: str, logger):
             "bundle_version": bundle_version
         }
     }
-    # TODO (mbaumann) Ensure webhooks are only delivered over verified HTTPS (unless maybe when running a test)
     callback_url = subscription['callback_url']
-    response = requests.post(callback_url, json=payload)
+
+    assert urlparse(callback_url).scheme in {"http", "https"}
+    if os.environ["DSS_DEPLOYMENT_STAGE"] not in {"dev", "staging"}:
```

Comment:
Is the intention here to ensure that for production deployments the address type is global and the scheme is "https"? If so, can/should we test for that directly using `os.environ["DSS_DEPLOYMENT_STAGE"] in {"prod"}`?
One reason I ask is that at UCSC we use custom stage names extensively (e.g. in my environment "DSS_DEPLOYMENT_STAGE" is set to "mbaumann", the custom stage name to which I deploy often), which then causes test_indexer.py to fail.
I suppose eventually/ideally there would a specific configuration setting indicating if this webhook validation should be performed, yet I don't think that is a required at this point.

Reply:
Yes, I will change it to read `{"prod"}`. The difficulty is that we have not "officially" deployed prod yet, so we just need to nail down that we will in fact name it `prod`.

Reply:
Changed to compare to "prod".